west.yml

```diff
@@ -237,6 +237,11 @@ manifest:
       path: modules/hal/stm32
       groups:
         - hal
+    - name: hal_tdk
+      revision: 16eaeeb54138348fb3227323d7d7c4b94e96ac3d
+      path: modules/hal/tdk
+      groups:
+        - hal
```

Comment:
The HAL commit should be first in the series, otherwise this PR isn't bisectable

Reply:
Thanks for this comment. I reorder my commits to have the HAL commit first.